### ISSUE TITLE
SkyrimTogehter AE crashing when loading in a save

### ISSUE THREAD

**Issue by Space2Code** [tiltedphoques/TiltedEvolution#499]
Every time I try to load in to a save the game crashes
https://youtu.be/kY8SKqp4Pg4

**Comment by mdio** [comment]
I can confirm this as well.
It started happening only after the game was updated to 1.6.629.
I had to update Address Library for SKSE Plugins to version 6, otherwise I would get "Failed to load Skyrim Address Library." on startup.
When I continue with the latest save I'm in the game for half a second, then it freezes and crashes.
Loading another save or starting a new game crashes in the loading screen already.

I tried enabling Papyrus logging by following [instructions](https://www.technewstoday.com/enable-and-access-skyrim-crash-logs/) but didn't get any log.
I only got a [cef_debug.log](https://github.com/tiltedphoques/TiltedEvolution/files/9578841/cef_debug.log) of a crash, I hope that helps. Otherwise let me know what I can do to help debug this issue if anything.
